test/image-blueprints/rhel92-source-fake-next-minor.toml

```diff
@@ -0,0 +1,25 @@
+name = "rhel-9.2-microshift-source-fake-next-minor"
```

Comment:
I might have just called this "rhel-9.2-microshift-next-minor" or "rhel-9.2-microshift-fake-next-minor", but we haven't really talked about naming conventions and that's a nit so let's not worry about it for now.